packages/types/src/payment-types.ts

```diff
@@ -54,12 +32,60 @@ export interface IFeeReferenceBasedCreationParameters extends IReferenceBasedCre
 }
 
 /** Parameters to create a request with "any to erc20" payment network */
-export interface IAnyToErc20CreationParameters extends IFeeReferenceBasedCreationParameters {
-  network?: string;
+export interface IAnyToErc20CreationParameters extends ICreationParametersAnyToAny {
   acceptedTokens?: string[];
-  maxRateTimespan?: number;
 }
 
+/** Interface to create a payment network  */
+export interface IPaymentNetworkCreateParameters<T = any> {
```

Comment:
do we still need the default `any`?

Reply:
I have actually deprecated this interface in the meantime @benjlevesque